scripts/sizeSnapshot/webpack.config.js

```diff
@@ -139,7 +139,7 @@ async function getWebpackEntries() {
     ...coreComponents,
     {
       id: '@material-ui/utils',
-      path: 'packages/mui-utils/build/esm/index.js',
+      path: 'packages/mui-utils/build/index.js',
```

Comment:
needs to change to a correct build path since all functions have been moved to subfolders in #39882